VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json

```diff
@@ -604,7 +604,9 @@
     "hearing-controls": "Rheolyddion y gwrandawiad",
     "consultation-controls": "Rheolaethau Ymgynghori",
     "show-chat-window": "Dangos y rhestr negeseuon",
-    "hide-chat-window": "Cuddio’r rhestr negeseuon"
+    "hide-chat-window": "Cuddio’r rhestr negeseuon",
+    "share-video-evidence": "Share video or audio",
```

Comment:
Is a ticket raised for these translations?

Reply:
Have you requested the translations? We have started to track the translations that need updating in a ticket when we dont have them but still want the ticket to be merged

Reply:
ah thanks bad copy and paste!